corva/models/scheduled.py

```diff
@@ -2,41 +2,52 @@
 
 import itertools
 from datetime import datetime
-from typing import List, Optional
+from typing import List, Literal, Optional
 
 import pydantic
 
-from corva.models.base import BaseContext, BaseEvent, CorvaBaseModel
-
-
-class ScheduledEventData(CorvaBaseModel):
-    type: Optional[str] = None
-    collection: Optional[str] = None
-    cron_string: str
-    environment: str
-    app: int
-    app_key: str
-    app_connection_id: int = pydantic.Field(alias='app_connection')
-    app_stream_id: int = pydantic.Field(alias='app_stream')
-    source_type: str
-    company: int
-    provider: str
-    api_url: Optional[str] = None
-    api_key: Optional[str] = None
-    schedule: int
-    interval: int
+from corva.models.base import BaseContext, BaseEvent
+
+
+class ScheduledEvent(BaseEvent):
+    asset_id: int
```

Comment:
Can't some of these fields go on the BaseEvent to deduplicate?

Reply:
the only common field for three events is asset_id. maybe moving one field to base class is not sufficient as it will lead to less code readability (jumping between base and child models)

Reply:
OK, I thought there were more shared fields than that. Sounds good!